### PR TITLE
added figsize args to surf plotting functions

### PR DESCRIPTION
**Pull request by JohnGriffiths**

### CODE REVIEW

File: nilearn/plotting/surf_plotting.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ def plot_surf(surf_mesh, surf_map=None, bg_map=None,
               hemi='left', view='lateral', cmap=None,
               avg_method='mean', threshold=None, alpha='auto',
               bg_on_data=False, darkness=1, vmin=None, vmax=None,
-              title=None, output_file=None, **kwargs):
+              title=None, output_file=None,figsize=None, **kwargs):
```
Comment (juhuntenburg) [inline nilearn/plotting/surf_plotting.py]: Space missing before "figsize"

File: nilearn/plotting/surf_plotting.py [hunk 2]
```diff
@@ -369,7 +369,7 @@ def plot_surf_stat_map(surf_mesh, stat_map, bg_map=None,
                        hemi='left', view='lateral', threshold=None,
                        alpha='auto', vmax=None, cmap='coolwarm',
                        symmetric_cbar="auto", bg_on_data=False, darkness=1,
-                       title=None, output_file=None, **kwargs):
+                       title=None, output_file=None, figsize=None,**kwargs):
```
Comment (juhuntenburg) [inline nilearn/plotting/surf_plotting.py]: Space missing after comma

File: nilearn/plotting/surf_plotting.py [hunk 3]
```diff
@@ -464,7 +464,7 @@ def plot_surf_stat_map(surf_mesh, stat_map, bg_map=None,
                         threshold=threshold, cmap=cmap,
                         alpha=alpha, bg_on_data=bg_on_data, darkness=1,
                         vmax=vmax, title=title, output_file=output_file,
-                        **kwargs)
+                        figsize=figsize,**kwargs)
```
Comment (juhuntenburg) [inline nilearn/plotting/surf_plotting.py]: Space missing after comma

File: nilearn/plotting/surf_plotting.py [hunk 4]
```diff
@@ -473,7 +473,7 @@ def plot_surf_roi(surf_mesh, roi_map, bg_map=None,
                   hemi='left', view='lateral', alpha='auto',
                   vmin=None, vmax=None, cmap='coolwarm',
                   bg_on_data=False, darkness=1,
-                  title=None, output_file=None, **kwargs):
+                  title=None, output_file=None, figsize=None,**kwargs):
```
Comment (juhuntenburg) [inline nilearn/plotting/surf_plotting.py]: Space missing after comma

File: nilearn/plotting/surf_plotting.py [hunk 5]
```diff
@@ -576,6 +576,6 @@ def plot_surf_roi(surf_mesh, roi_map, bg_map=None,
                         cmap=cmap, alpha=alpha, bg_on_data=bg_on_data,
                         darkness=darkness, vmin=vmin, vmax=vmax,
                         title=title, output_file=output_file,
-                        **kwargs)
+                        figsize=figsize,**kwargs)
```
Comment (juhuntenburg) [inline nilearn/plotting/surf_plotting.py]: Space missing after comma

**Comment by GaelVaroquaux** [comment]
I believe that you can pass in a figure as an argument. This would be the
right way of solving the original problem, and not adding an argument.

**Comment by KamalakerDadi** [comment]
>I believe that you can pass in a figure as an argument.

This means that we can pass user defined figure directly as a kwarg or add an argument "figure" rather than figsize like other plotting functions. If figure is None we take default otherwise user given figure ?

**Comment by GaelVaroquaux** [comment]
Comming back to this PR: I was wrong (my apologies), you cannot pass the figure.

However, I would prefer if we could pass a figure, rather than a figsize, because this is how it's implemented in the volume plotting functions.

**Comment by kchawla-pi** [comment]
Is this alive? If not, please close it by this weekend.

**Comment by tsalo** [comment]
It seems like this is not alive. Closing.

**Pull request closed without merging**